proto/cosmos/crisis/module/v1/module.proto

```diff
@@ -0,0 +1,12 @@
+syntax = "proto3";
+
+package cosmos.crisis.module.v1;
+
+import "cosmos/app/v1alpha1/module.proto";
+
+// Module is the config object of the crisis module.
+message Module {
+  option (cosmos.app.v1alpha1.module) = {
+    go_import: "github.com/cosmos/cosmos-sdk/x/crisis"
+  };
```

Comment:
expecting to see fee collector name here. looks like it's in the generated code

Reply:
Yeah I forgot to push sorry